### PR TITLE
(#155) add information box when there is no storage/configmaps/secrets

### PR DESCRIPTION
**Pull request by cuddlehub**

### CODE REVIEW

**Review by kevinlinglesas — COMMENTED** [review]
Changes look ok to me, but I defer to @sasjowood for authoritative review/approval.

File: deployment_report/templates/viya_deployment_report.html.j2 [hunk 1]
```diff
@@ -299,14 +308,22 @@
...
   {# Configurations: Content #}
 
-  {# Cluster Overview: Configmaps Accordion #}
-  {% set kind_name = "Configmaps" %}
   <div class="jq-accordion">
-    <h2>Configmaps</h2>
-    <div>
-      {% for name, info in report_data.kubernetes.configMaps.items() %}
+  {# Configurations: Configmaps Accordion #}
+  <h2>Configmaps</h2>
```
Comment (sasjowood) [inline deployment_report/templates/viya_deployment_report.html.j2:313]: Let's change `<h2>Configmaps</h2>` to `<h2>ConfigMaps</h2>` to better align with the way k8s formats the name.

File: deployment_report/templates/viya_deployment_report.html.j2 [hunk 2]
```diff
@@ -299,14 +308,22 @@
...
+  <div>
+  {% set kind_name = "Configmaps" %}
+  {% if report_data.kubernetes.configMaps|length == 0 %}
+    <div class="notification-box info-box">
+      <i class="fa fa-info-circle notification-box-icon" aria-hidden="true"></i>
+      <div class="notification-box-text margin-block-end-1em">
+        No configMaps were found in namespace [{{ report_data.kubernetes.namespace }}].
```
Comment (sasjowood) [inline deployment_report/templates/viya_deployment_report.html.j2:320]: Let's change `configMaps` to `ConfigMaps` to better align with the way k8s formats the name.

File: deployment_report/templates/viya_deployment_report.html.j2 [hunk 3]
```diff
@@ -335,61 +352,76 @@
...
-                {% set ns.secretvalue = '' %}
-                {% for i in value2 %}
-                  {% set ns.secretvalue = ns.secretvalue ~ '<a class="nested-accordions-rel-link" href="#pod-secrets_' ~ i ~ '">' ~ mybr ~ '&bull; ' ~ i ~ '</a>' %}
-                {% endfor %}
-                {% if ns.secretvalue != '' %}
-                  {% set ns.keyrefurl = ns.keyrefurl ~ mybr ~ "Referenced pods containers:" ~ ns.secretvalue ~ mybr ~ mybr %}
+      <div class="notification-box info-box">
+        <i class="fa fa-info-circle notification-box-icon" aria-hidden="true"></i>
+        <div class="notification-box-text margin-block-end-1em">
+          No secrets were found in namespace [{{ report_data.kubernetes.namespace }}].
```
Comment (sasjowood) [inline deployment_report/templates/viya_deployment_report.html.j2:371]: Let's change `secrets` to `Secrets` to better align with how k8s formats the name.

**Review by sasjowood — CHANGES_REQUESTED** [review]
Overall, the changes look great. There are a few places below where we can update the formatting of a k8s resource name to better align with how they display things. Other than those small changes, this looks ready to merge.

**Pull request merged**